Settings

evanpacini opened the pull request:
Added the reverse polarity(minus becomes plus and vice versa) and remap speeds option(switch engines).

Resolves #23

evanpacini committed:
Added Settings

Added remap speed and reverse polarity options in HTML.

evanpacini committed:
Basic styling for settings

I made some quick (very basic) styling for the settings section.

evanpacini committed:
Added reverse polarity and remap speed functions

I added both functions in js, however this is only for joystick controls. I don't know if it's really needed for keyboard at all.

evanpacini committed:
Seperate reverse polarity tuning

I made the reverse polarity setting seperate (you could have plugged only one motor wrongly).

evanpacini committed:
Seperate reverse polarity tuning

I made the reverse polarity setting seperate (you could have plugged only one motor wrongly).

KaspervanM commented:
I have tested it and I will not yet approve it. Many thing are broken with this enhancement/bugfix. 
It only seems to work with the joystick in tank mode. I couldn't test car mode because I couldn't switch between modes anymore. Also, the function completely doesn't change the keys serial output.

The joystick works perfectly though, but that isn't the only thing on the interface.

evanpacini committed:
Bugfix drive mode

evanpacini committed:
Keyboard settings beta

I made a begining to being able to apply settings to keyboard control.

evanpacini committed:
Settings now working on keyboard

Now the settings should be applied on keyboard too. However, I wasn't able to test if it works.

evanpacini committed:
Added tooltips

I added some tooltips to make the settings more understandable.

evanpacini committed:
Tooltip styling

Some styling for the tooltips I made.
